components/googleMap/googleMap.js

```diff
@@ -0,0 +1,306 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
```

Comment:
both seem to work 👍 

tried out here: https://codesandbox.io/s/KrmEjrmpz

would be nice if the project sticks to one way of using it though across all the components.